src/Components/AddComment.jsx

```diff
@@ -0,0 +1,58 @@
+import {useState} from "react"
+import { PostComment } from "../Api";
+
+function AddComment({review_id, setCommentsList}){
+    
+
+    const [userName, setUserName] = useState("");
+    const [body, setBody] = useState("");
+
+    const handleSubmit = (e) => {
```

Comment:
Just personal preference here, but I always like to see `event` rather than `e`, cos I'm an idiot and can never remember how my app slots together and it reminds me that events come from the user and responses come from an api...